### DISCUSSION TITLE
Why does the localhost server not work?

### DISCUSSION THREAD

**Discussion by Hashimk2** [HasangerGames/suroi#436] [Ideas]
when you click the servers on suroi.io,  the top one is localhost server, but when i click it and play solo, squads or duos, it says, 'error joining game'
![Screenshot (758)](https://github.com/user-attachments/assets/6b77c17f-bd85-4e4f-80b4-cd4148a5be74)

**Comment by kidwaih2** [comment]
There was a mistake, the localhost server is not supposed to be there. Just play in any other servers (you can close this now)

**Reply by Crimson-Haze** [reply]
What does it do?
